concepts/05 UI Components/DataGrid/70 Custom Keyboard Navigation/15 Override and Create Keys.md

```diff
@@ -0,0 +1,105 @@
+The [onKeyDown](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onKeyDown) event handler allows you to track keys used while the UI component is in focus. You can use it to override the default keyboard support shortcuts, implement your own shortcuts, or extend existing ones.
```

Comment:
When you mentioned this event handler, does it mean that it's a definite event handler, not just any event handler? Can we say it's a predefined event handler? I just have some doubts regarding the article choice.

I'm not changing it for now, while I do not have full understanding of the current context.

> the UI component is in focus

the UI component has focus, not "is in focus". This looks like a bit of Runglish to me.

>  You can use **it** to override the

Use whom? The mentioned handler, the component, or focus? It's completely unclear but I suspect that you meant the event handler.

> default keyboard **support** shortcuts

I'm unsure what you've meant here.

```suggestion
The [onKeyDown](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onKeyDown) event handler allows you to track pressed keystrokes while the UI component has focus. You can use the event handler to override the default keyboard shortcuts, implement custom keystrokes, or extend existing ones.
```

Reply:
It's predefined event handler. So this is why I used the definite article.

Reply:
OK. This makes sense, thank you. In this case, I believe the last version of this wording is probably fine, provided that you agree with the rest.